xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -164,6 +164,7 @@
     </property>
     <property>
       <async_context>
+        <value>doc.reference</value>
```

Comment:
Doesn't this make caching pretty much useless? Note that this cache is small (100 entries by default).